pkg/client/clientset/versioned/clientset.go

```diff
@@ -19,6 +19,7 @@ package versioned
 import (
 	"fmt"
 
+	kubeflowv1 "github.com/kubeflow/training-operator/pkg/client/clientset/versioned/typed/pytorch/v1"
```

Comment:
@terrytangyuan here is an example of the conflicting import mentioned in this [comment](https://github.com/kubeflow/training-operator/issues/1499#issuecomment-987306746)